src/virtual-wan/azext_vwan/profiles.py

```diff
@@ -7,3 +7,4 @@
 
 CUSTOM_VWAN = CustomResourceType('azext_vwan.vendored_sdks.v2020_05_01', 'NetworkManagementClient')
 CUSTOM_VWAN_2021_03_01 = CustomResourceType('azext_vwan.vendored_sdks.v2021_03_01', 'NetworkManagementClient')
+CUSTOM_VWAN_2021_08_01 = CustomResourceType('azext_vwan.vendored_sdks.v2021_08_01.v2021_08_01', 'NetworkManagementClient')
```

Comment:
Please change `azext_vwan.vendored_sdks.v2021_08_01.v2021_08_01` to `azext_vwan.vendored_sdks.v2021_08_01` which is because that you added duplicated 
 folder `v2021_08_01` in vendored_sdks [vendored_sdks/v2021_08_01/v2021_08_01/__init__.py]